task10-vue/src/components/IslandSection.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <article class="article article--islands">
+    <div class="article__container article__container--islands">
+      <h3 class="article__title article__title--island">Islands</h3>
+      <a href="#" class="article__link article__link--islands">See all</a>
+    </div>
+    <div class="article__flex article__flex--islands">
+      <IslandCard
+        v-for="island in islands"
+        :key="island.name"
```

Comment:
"key" attribute should be unique in order to increase performance while rendering components, "v-for" provides an index attribute you could use here "v-for="(island, index) in islands"